src/indexer.c

```diff
@@ -572,6 +589,25 @@ static DocumentIndexer *NewDocumentIndexer(const char *name, int options) {
   return indexer;
 }
 
+static void FreeDocumentIndexer(DocumentIndexer *indexer) {
```

Comment:
@mnunberg we already have in this file GetDocumentIndexer are you sure you want to break this convention?

Reply:
DropDocumentIndexer, like GetDocumentIndexer, makes sense, because it's operating on a global list, takes a name, which may or may not correspond to an actual indexer.

Indexer_Run, which has a specific indexer as its object, has a different naming convention. Likewise, Indexer_Free.

Reply:
@mnunberg but here indexer_free should be static so its another convention isn't it?

Reply:
yeah, indexer_free is static, like indexer_run, because it's not used externally.

Reply:
Ok, changed